src/ui/public/flot-charts/jquery.flot.pie.js

```diff
@@ -55,8 +55,9 @@ More detail and specific examples can be found in the included HTML file.
 
 */
 
-(function($) {
+import { i18n } from '@kbn/i18n';
 
+(function($) {
```

Comment:
Sometimes I don't get git.... That's perfectly fine detected as a simple adding of one line above that in all other files, but here it thinks the line got removed and readded...